soda-consumer-scala/src/main/scala/com/socrata/soda2/consumer/impl/RowDecoder.scala

```diff
@@ -59,14 +59,15 @@ class RowDecoder(datasetBase: URI, schema: Map[ColumnName, SodaType]) extends (J
 
 object RowDecoder {
   def cookSchema(rawSchema: Map[ColumnName, String]) =
-    rawSchema.mapValues(conversionForType).toMap
+    rawSchema.mapValues(conversionForType)
```

Comment:
That `.toMap` isn't superfluous; `mapValues` produces a view and the `toMap` forces it back to an actual fully-evaluated collection.